Apps/Src/ReadTritium.c

```diff
@@ -6,7 +6,7 @@ void Task_ReadTritium(void* p_arg) {
 
 	OS_ERR err;
 	
-	car_state_t *car_state = (car_state_t *) p_arg;
+	//car_state_t *car_state = (car_state_t *) p_arg;

```

Comment:
Why not just delete instead of commenting out in this file?

Reply:
Done